### ISSUE TITLE
 Feature Request: Add Continent Dropdown Component

### ISSUE THREAD

**Issue by Noor-infoarc** [venkatmcajj/react-country-state-city#13]
Would it be possible to add a continent dropdown component to the library? 

It would be beneficial to include a continent dropdown, providing a user-friendly way to select continents directly. This could be useful for applications needing geographic data categorization or regional selection without requiring more detailed country data.

**Comment by venkatmcajj** [comment]
Yes it's Added, Install the latest version then you can use the continent dropdown.
